client_test/directfund_test.go

```diff
@@ -31,7 +31,7 @@ func directlyFundALedgerChannel(t *testing.T, alpha client.Client, beta client.C
 		AppDefinition:     types.Address{},
 		AppData:           types.Bytes{},
 		ChallengeDuration: big.NewInt(0),
-		Nonce:             rand.Int63(),
```

Comment:
Am I right in thinking the issue is that (with very high probability) the nonce was simply too large to be expressed as a 48 bit integer? Presumably the data type itself isn't the real issue, since after the fix we are generating a smaller random number and then casting to a 64 bit int anyway?